### DISCUSSION TITLE
帕鲁应用在修改配置文件后，配置文件会还原

### DISCUSSION THREAD

**Discussion by faruzanCN** [1Panel-dev/1Panel#4119] [Show and tell]
### 联系方式

1Panel首批赞助用户微信群LuvLetter

### 1Panel 版本

V1.10.1-lts

### 问题描述

修改/opt/1panel/apps/palworld/palworld/data/Pal/Saved/Config/LinuxServer/PalWorldSettings.ini文件任意内容后，重启对应docker就会将此文件恢复为默认状态（还原），如将文件内容DeathPenalty的值由All修改为None。

### 重现步骤

见问题描述

### 期待的正确结果

重启docker不会将配置文件还原

**Comment by igophper** [comment]
这个是帕鲁服务器的原因，和1panel没关系。你想修改配置文件，应该先停止docker再去修改，保存之后再启动docker

**Reply by Tomori77** [reply]
停止后修改仍然会变回默认，已经让折磨了

**Comment by Tomori77** [comment]
想知道你解决了没有，我现在也是这个情况

**Reply by faruzanCN** [reply]
没解决  官方说是我操作不对，建议继续提is
